src/coreComponents/dataRepository/Group.hpp

```diff
@@ -854,6 +854,13 @@ class Group
   ///@}
   //END_SPHINX_INCLUDE_REGISTER_WRAPPER
 
+  /**
+   * @brief Append a levelCondition and a log description to the description of the wrapped object
+   * @param levelCondition The level condition to append
+   * @param logDescription The log description to append
+   */
+  void appendLogLevelDescription( string_view levelCondition, string_view logDescription );
```

Comment:
Is there a way, instead of having now 4 `public` member functions and 2 attributes directly related, and a entry in the `viewKeysStruct` to gather all the behaviors related to the log into a dedicated `class`?
I personally do not want to add more and more stuff into `Group` which is already big enough (+/- 120 `public` member functions IIRC).

Reply:
A bit off-topic comment : I don't know if it would help, but in a long-term effort. I would say that if we use a similar approach with the group's wrappers with a `WrapperContainers`, we could remove around 36 members (including const/non-const), allowing to clarify that methods like "size()" are related to Wrappers.